docs/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernete's community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+[Kubernete's community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
+[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Onprem
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
```

Comment:
In case of code changes we should mention that we require unit-tests. 
Maybe need to add guidelines to the reviewers as well.

Reply:
Indeed! 

I will mention this and I will also add a `Reviewers checklist` to this template. Eventually, once we have the guidelines docs, we can also link them here.